HangFire.Autofac/HangfirePerLifetimeScopeConfigurer.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Autofac;
+
+namespace Hangfire
+{
+    public static class HangfirePerLifetimeScopeConfigurer
+    {
+        public static void Configure(IContainer container)
```

Comment:
Maybe it is better to leverage the new `GlobalConfiguration` class and implement the `UseScopedAutofacActivator` (or so) extension method for `IGlobalConfiguration` interface? In this case configuration will be straight:

```csharp
GlobalConfiguration.Configuration
    .UseSqlServerStorage()
    .UseScopedAutofacActivator();
```